docs/user/reference/general/register-custom-pings.md

```diff
@@ -0,0 +1,75 @@
+# Registering custom pings
+
+After defining [custom pings](../../user/pings/custom.md) The Glean SDK build generates code from `pings.yaml` in a `Pings` object, which must be instantiated so Glean can send pings by name.
+
+## Loading custom ping metadata into your application or library
+
+{{#include ../../../shared/tab_header.md}}
+
+<div data-lang="Kotlin" class="tab">
+
+In Kotlin, this object must be registered with the Glean SDK from your startup code before calling `Glean.initialize`
+(such as in your application's `onCreate` method or a function called from that method).
+
+```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Pings
+
+override fun onCreate() {
+    Glean.registerPings(Pings)
+
+    Glean.initialize(applicationContext, uploadEnabled = true)
+}
+```
+
+</div>
+
+<div data-lang="Swift" class="tab">
+
+In Swift, this object must be registered with the Glean SDK from your startup code before calling `Glean.shared.initialize`
+(such as in your application's `UIApplicationDelegate` [`application(_:didFinishLaunchingWithOptions:)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application) method or a function called from that method).
+
+```swift
+import Glean
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        Glean.shared.registerPings(GleanMetrics.Pings)
+
+        Glean.shared.initialize(uploadEnabled = true)
+    }
+}
+```
+
+</div>
+
+<div data-lang="Python" class="tab">
```

Comment:
From this text it actually seems Python does not provide the `registerPings` API?

Reply:
It does have one, but it's implicitly called by the other code.

Reply:
Hm, this is weird... but ok.